rust/cbork-utils/src/decode_helper.rs

```diff
@@ -177,4 +218,12 @@ mod tests {
         // Should print out the error message with the location of the error
         assert!(result.is_err());
     }
+
+    #[test]
+    fn test_decode_any_seq() {
```

Comment:
Didn't get, what we are testing with this test case ?
seems like its a redundant test case